main/src/main/scala/sbt/Keys.scala

```diff
@@ -376,6 +376,7 @@ object Keys {
   val packagedArtifacts = taskKey[Map[Artifact, File]]("Packages all artifacts for publishing and maps the Artifact definition to the generated file.").withRank(CTask)
   val publishMavenStyle = settingKey[Boolean]("Configures whether to generate and publish a pom (true) or Ivy file (false).").withRank(BSetting)
   val credentials = taskKey[Seq[Credentials]]("The credentials to use for updating and publishing.").withRank(BMinusTask)
+  val allCredentials = taskKey[Seq[Credentials]]("Aggregated credentials.").withRank(DTask)
```

Comment:
I think this needs either a more descriptive info string or to be `private[sbt]`. As it stands, I wouldn't be sure whether or not I should be updating `credentials` or `allCredentials`.

Reply:
I'll update the text.

Reply:
9755234